### DISCUSSION TITLE
Can we query based on a single point (one lat one long)?

### DISCUSSION THREAD

**Discussion by priyapatel1** [earthaccess-dev/earthaccess#478] [Q&A] [answered]
I don't want a bounding box, I want to query based on one point - is that possible?

**Comment by mfisher87** [comment, marked as answer]
Hi @priyapatel1 , thanks for your question. It is possible! Please use the `point` argument to `search_data()`, e.g. `earthaccess.search_data(point=(lon, lat))`.

https://earthaccess.readthedocs.io/en/latest/user-reference/granules/granules-query/#earthaccess.search.DataGranules.point

Can you please select "Mark as answer" if this helps?

**Reply by mfisher87** [reply]
Note that this is _not_ documented here, where I looked first: https://earthaccess.readthedocs.io/en/latest/user-reference/api/api/#earthaccess.api.search_data

I think we could improve that.
